package.json

```diff
@@ -102,7 +102,8 @@
     "react-dom": "^16.13.0",
     "react-router-dom": "^6.2.1",
     "styled-components": "^6.1.8",
-    "styled-css-grid": "^1.2.1"
+    "styled-css-grid": "^1.2.1",
+    "@reduxjs/toolkit": "^1.9.5"
```

Comment:
This should be in saga-injector package.json
is better if you move it to devDep and peer depencency